xml/ses_glossary.xml

```diff
@@ -52,20 +117,79 @@
     </para>
    </glossdef>
   </glossentry>
-  <glossentry><glossterm>Monitor node, MON</glossterm>
+  <glossentry><glossterm>CRUSH rule</glossterm>
+   <glossdef>
+    <para>
+     The CRUSH data placement rule that applies to a particular pool(s).
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>&dashboard;</glossterm>
+   <glossdef>
+    <para>
+     A built-in web-based &ceph; management and monitoring application to
+     administer various aspects and objects of the cluster. The dashboard is
+     implemented as a &mgr; module.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>&drvgrps;</glossterm>
+   <glossdef>
+    <para>
+     &drvgrps; are a declaration of one or more OSD layouts that can be mapped
+     to physical drives. An OSD layout defines how &ceph; physically allocates
+     OSD storage on the media matching the specified criteria.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>&grafana;</glossterm>
+   <glossdef>
+    <para>
+     Database analytics and monitoring solution.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>&mds;</glossterm>
+   <glossdef>
+    <para>
+     &mds; or MDS is the &ceph; metadata software.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>&mon;</glossterm>
+   <glossdef>
+    <para>
+     &mon; or MON is the &ceph; monitor software.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>&mgr;</glossterm>
    <glossdef>
     <para>
-     A cluster node that maintains maps of cluster state, including the monitor
-     map, or the OSD map.
+     &mgr; or MGR is the &ceph; manager software, which collects all the state
+     from the whole cluster in one place.
+    </para>
+   </glossdef>
+  </glossentry>
+  <glossentry><glossterm>Node</glossterm>
+   <glossdef>
+    <para>
+     Any single machine or server in a Ceph cluster.
     </para>
    </glossdef>
   </glossentry>
   <glossentry><glossterm>OSD</glossterm>
    <glossdef>
     <para>
-     Depending on context, <emphasis>Object Storage Device</emphasis> or
-     <emphasis>Object Storage Daemon</emphasis>. The
-     <command>ceph-osd</command> daemon is the component of &ceph; that is
+     <emphasis>Object Storage Device</emphasis>: A physical or logical storage
+     unit.
```

Comment:
Are we deprecating the "daemon" meaning?

Reply:
It's now below :)

Reply:
Aha! OK.

If I read through these in advance, before commenting, I find that 2nd time around I was expecting some things and my eye skips over them, resulting in me forgetting to comment.

That's my excuse and I'm sticking to it.